app/src/lib/git/worktree.ts

```diff
@@ -3,6 +3,7 @@ import * as Path from 'path'
 import * as FSE from 'fs-extra'
 
 import { git } from './core'
+import { v4 as uuid } from 'uuid'
```

Comment:
I didn't use our internal wrapper for uuid because electron has since fixed the performance issues that motivated its creation.